ui/apps/platform/cypress/integration/systemHealth/clusters.test.js

```diff
@@ -366,7 +419,7 @@ describe('System Health Clusters subset 1 Healthy', () => {
 
         const cardTitle = 'Cluster status';
 
-        cy.get(getCardHeaderDescendantSelector(cardTitle, 'span:contains("healthy")'));
+        cy.get(getCardHeaderDescendantSelector(cardTitle, 'div:contains("1 healthy")'));
```

Comment:
red flag: we are hard-coding the number we expect. Are we mocking it to be sure it will always be 1 ?

Reply:
Yes, these are from fixtures.